fastlane/lib/fastlane/actions/oclint.rb

```diff
@@ -90,6 +92,22 @@ def self.run(params)
         return Action.sh(command)
       end
 
+      # return a proper regex object if regex string is single-quoted
+      def self.ensure_regex_is_not_string!(regex)
+        unless regex.nil?
+          if regex.kind_of?(String)
+            safer_eval = Thread.new do
+              $SAFE = 1
```

Comment:
Regexp requires that the string be single-quoted, otherwise you have to extra-escape things.